drivers/platform/stm32/stm32_spi.c

```diff
@@ -46,6 +46,52 @@
 #include "no_os_delay.h"
 #include "no_os_alloc.h"
 
+/**
+ * @brief spi_table contains the pointers towards the SPI buses
+*/
+static void *spi_table[8];
+
+/**
+ * @brief STM32 SPI bus memory initializer
+*/
+int stm32_spibus_init(uint32_t bus_number, struct no_os_spi_init_param *param)
```

Comment:
One important remark here.
It seems your implementation doesn't have anything to do with stm32.
I think this could somehow be moved to no_os_spi.c, could you please investigate that ?
That way there wouldn't be a need to implement it in each [platform]_spi.c

Reply:
Yes, it would make sense.